### PR TITLE
Modified Governance page

### PR DESCRIPTION
**Pull request by JustAnotherDevv**
Changes:

- Changed [generic realms voting link](https://app.realms.today/realms) to [helius vote realms link](https://realms.heliumvote.com/realms)
- Fixed inconsistent use of `subDAOs` vs `subDAOS` to just `subDAOs`
- Added distinction between instructions for old vs new voting interface
- Removed `Compressed NFTs` page since it's empty and information about compressed NFTs has been moved to [Hotspots on Solana](https://docs.helium.com/solana/rewardable-entites) page already

### CODE REVIEW

File: docs/architecture/solana/migration/exchange.mdx [hunk 1]
```diff
@@ -28,7 +28,7 @@ If an exchange already supports SPL tokens, follow the steps below.
 :::tip SPL token Support
 
 Please refer to the Solana Documentation on how to
-[Add Solana to Your Exchange](https://docs.solana.com/integrations/exchange) for details.
+[Add Solana to Your Exchange](https://solana.com/docs/more/exchange) for details.
```
Comment (jthiller) [inline docs/architecture/solana/migration/exchange.mdx]: This change is already handled on main. Let's rebase this branch to pull in those changes.

File: sidebarsDocs.js
Comment (jthiller) [inline sidebarsDocs.js:1]: Let's keep syntax formatting out of PRs related to content changes.

Comment (JustAnotherDevv) [reply sidebarsDocs.js:1]: Fixed

**Review by waveform06 — CHANGES_REQUESTED** [review]
the Compression nfts page is going to get updated so no need to delete, and no navigation to fix
We've moved away from Realms now to use a deployed instance of 
https://modular-governance.helium.com/
at https://heliumvote.com/
And prefer subnetworks to subDAOs

**Comment by JustAnotherDevv** [comment]
> the Compression nfts page is going to get updated so no need to delete, and no navigation to fix We've moved away from Realms now to use a deployed instance of https://modular-governance.helium.com/ at https://heliumvote.com/ And prefer subnetworks to subDAOs

Will change subDAOs to subnetworks.

However I haven't seen https://modular-governance.helium.com/ mentioned anywhere in documentation before:
-  Is there a need to set up new pages for it with information and instructions how to use it? I can contribute to this if needed.
- If Realms are no longer used for governance should I remove pages related to them to help avoid confusion for new users?

**Review by jthiller — CHANGES_REQUESTED** [review]
Please rebase this PR to clean up the commit history for cleaner review.

**Pull request closed without merging**